package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ipguk/react-ui",
-  "version": "2.23.0",
+  "version": "2.23.0-2",
```

Comment:
You know you can install npm dependency from a branch/commit right? Just checking in case you weren't aware. Saves you having to publish a new version tp npm each time you want to test your changes in another app.